src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[],
+  isTemp?: Todo | null,
+  onRemoveTodo: (id: number) => void,
+  onCompletedChange: (value: Todo) => void
+};
+
+export const TodoList: React.FC<Props> = React.memo(
+  ({
+    todos, isTemp, onRemoveTodo, onCompletedChange,
+  }) => {
+    return (
```

Comment:
can omit

Reply:
https://github.com/mate-academy/react_todo-app-add-and-delete/pull/327/files#r1124829266